application/tests/common_test.yaml

```diff
@@ -0,0 +1,109 @@
+suite: Common
+
+templates:
+  # TODO: test all templates
+  # - '*.yaml'
+  - deployment.yaml
+  - ingress.yaml
+  - service.yaml
+  - extraobjects.yaml
+
+values:
+  - ../values-test.yaml
```

Comment:
@aslafy-z maybe this here will never work in the CI because of `mv`s there... Any idea how to use the test values?